migration/src/audio.rs

```diff
@@ -48,8 +144,15 @@ impl DrsRes {
             tries += 1;
         }?)
     }
+    /// Retrieves the content URL associated with a DRS resource
+    fn get_url(&self) -> String {
+        self.canonical_object.get_first_object_pid()
+    }
 }
 
+/// Represents annotation info associated with a single word
+///
+///
 #[derive(Serialize, Deserialize, Clone, Debug)]
 struct AudioAnnotationRow {
```

Comment:
is there a reason this is named Row if it represents a single word?

Reply:
Good catch. Does this look better?